add `extends` configuration variable (#61)
gzuidhof/tygo: README.md

    # in the output.
    exclude_files:
      - "private_stuff.go"

>     # Package that your type classes should extend. This is useful when

Reviewer: ```suggestion
    # Package that the generates Typescript types should extend. This is useful when
```
